game-two/Sources/App/Core/Models/Friendly/Player/Player.cs

```diff
@@ -205,8 +225,7 @@ public void GetJoystickStatus(bool state)
 				}
 				else if(!this.HangingStatus)
 				{
-					this.HangingStatus = true;
-					if(finalPos.x > 1000){						
+					if(finalPos.x > 1000){			
 						if(this.TentaculeArray[i].PositionRelativeToPlayer == "Right")	
```

Comment:
Chaine en constante

Reply:
Et plutot que : PositionRelativeToPlayer == "Right"
Préférer : this.TentaculeArray[i].IsPositionRight

Reply:
ok